Client/src/Components/product-reviews/review.jsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable camelcase */
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Helpful from "./Helpful";
+import RecommendProduct from "./RecommendProduct";
+import StarRating from "./StarRating";
+import Body from "./Body";
+import Response from "./Response";
+
+// Review function creates each individual review. Review function will call other functions to
+// delegate individual features within a review.
+// Each review will contain a rating, date, summary, body, helpfulness, recommend, photos, response,
+// and reviewer_name
+function Review({ review }) {
+  const {
+    rating, date, summary, body, helpfulness, recommend, photos, response, reviewer_name,
+  } = review;
+  const [helpfulCounter, setHelpCounter] = useState(helpfulness);
+  function incrementCounter() {
+    setHelpCounter(helpfulCounter + 1);
+  }
+  const realDate = new Date(date).toDateString().split(" ").slice(1)
+    .join(" ");
+  return (
+    <div>
+      <div id="rating">
+        <StarRating rating={rating} />
+      </div>
+      <div id="username" align="right"> </div>
```

Comment:
is the reviewer_name supposed to go into this div?

Reply:
I assume you're talking about the first div since there isnt an id attached to that one. Added id="review"